versionist.conf.js

```diff
@@ -1,32 +1,63 @@
-module.exports = {
+var _ = require('lodash');
+var execSync = require('child_process').execSync;
 
-  subjectParser: 'angular',
+var getAuthor = (commitHash) => {
+  var result = execSync(`git show ${commitHash}`, { encoding: 'utf8' });
```

Comment:
Using `git show --quiet --format="%an" [hash]` will return only the author's name, so no regex required.